library/pk_wrap.c

```diff
@@ -146,10 +154,9 @@ static int rsa_encrypt_wrap( void *ctx,
                                        ilen, input, output ) );
 }
 
-static int rsa_check_pair_wrap( const void *pub, const void *prv )
+static int rsa_check_pair_wrap( const mbedtls_pk_context *pub, const void *prv )
```

Comment:
While we're at it, we might reverse the order of the arguments; it's more in line with the rest of the interface that the context with respect to which the call is made is in the first argument.

Reply:
This would also require changing the order in rsa_alt_check_pair, eckey_check_pair and pk.c using it, and it would also be nice to change mbedtls_pk_check_pair along with tests for it.
Is that what you would like?

Reply:
@AndrzejKurek We cannot change anything public, unfortunately. @gilles-peskine-arm @mpg What's your take on this point? Is it worth changing the order, if we can't change it all the way through to the API?

Reply:
I prefer to keep the same order as the public `xxx_check_pair` functions. We can't change this order for backward compatibility.

Reply:
@gilles-peskine-arm Ok.

Reply:
Reverted my local changes - leaving this unchanged.